module/programs/helper.js

```diff
@@ -478,14 +491,16 @@ module.exports = class ProgramsHelper {
 				entitiesData.forEach((entity) => {
```

Comment:
@Prajwal17Tunerlabs what will happen if entitiesData is empty? Like if entity-management service gives empty data. Please check

Reply:
handled empty data validation